From e59e41c3d3221f62a11415e03a94bb8170ba89a8 Mon Sep 17 00:00:00 2001
From: Cristian Birsan <cristian.birsan@microchip.com>
Date: Tue, 25 Oct 2022 17:39:34 +0300
Subject: [PATCH 471/471] ARM: configs: at91: sama5: update config for KSZ
 ethernet switches

After merging KSZ pacthes from upstream, the kernel config used to enable
KSZ9477 DSA switch changed to a more generic form. This patch enables
it for SAMA5D2-ICP.

Signed-off-by: Cristian Birsan <cristian.birsan@microchip.com>
Tested-by: Sergiu Moga <sergiu.moga@microchip.com>
---
 arch/arm/configs/sama5_defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/sama5_defconfig b/arch/arm/configs/sama5_defconfig
index 83f1b35f29f7..5bb64be4fe70 100644
--- a/arch/arm/configs/sama5_defconfig
+++ b/arch/arm/configs/sama5_defconfig
@@ -85,8 +85,8 @@ CONFIG_BLK_DEV_SD=y
 # CONFIG_BLK_DEV_BSG is not set
 # CONFIG_SCSI_LOWLEVEL is not set
 CONFIG_NETDEVICES=y
-CONFIG_NET_DSA_MICROCHIP_KSZ9477=m
-CONFIG_NET_DSA_MICROCHIP_KSZ9477_SPI=m
+CONFIG_NET_DSA_MICROCHIP_KSZ_COMMON=m
+CONFIG_NET_DSA_MICROCHIP_KSZ_SPI=m
 # CONFIG_NET_VENDOR_3COM is not set
 # CONFIG_NET_VENDOR_ALACRITECH is not set
 # CONFIG_NET_VENDOR_AMAZON is not set
-- 
2.34.1

